packages/samples/headless-react/src/components/facet/facet.class.tsx

```diff
@@ -29,7 +29,6 @@ export class Facet extends Component<FacetProps> {
 
   componentDidMount() {
     this.unsubscribe = this.controller.subscribe(() => this.updateState());
-    this.controller.showMoreValues();
```

Comment:
When I originally added this call, I misunderstood that the reason that no results were showing was because no search was executed. Running this pointlessly executes a search, so I'm removing it.